check.sh

```diff
@@ -127,7 +127,7 @@ function cmd_fmt() {
 }
 
 function cmd_clippy() {
-    run cargo clippy --all-targets "${extraCargoArgs[@]}" -- \
+    run cargo clippy --all-targets "${extraCargoArgs[@]}" --features "itest/codegen-full" -- \
```

Comment:
For some reason, `--all-targets` causes `godot/default` to get enabled, and so far, I wasn't able to figure out why.

Reply:
Full codegen should be off by default, see [my other comment](https://github.com/godot-rust/gdext/pull/771#discussion_r1704015220).